### ISSUE TITLE
Sec508 Compliance Results Review for Remediation - Registrar

### ISSUE THREAD

**Issue by PaulKuykendall** [cisagov/manage.get.gov#2835]
### Issue description

We have received an initial report of findings for Section 508 compliance from our Trusted Tested in OAST and need to review these findings and create a remediation plan, including creation of all necessary dev tickets.

All report files are located on Google Drive [here](https://drive.google.com/drive/folders/1ixX8WijLH_D6oct_09noKlqehtZs-qgY).
[Summary Report](https://docs.google.com/document/d/19rsVRd6NkeX9-TV0p--E7twevEkx-pS7/edit?usp=drive_link&ouid=105479554793004732478&rtpof=true&sd=true)
[Complete Report](https://drive.google.com/file/d/1lEjoN7Xeeoi7atxuT2oHxJ6WN8NLpgL7/view?usp=drive_link) (pdf from html)

[Remediation Plan](https://docs.google.com/document/d/1L7TW2RdxAFUrWWLSw3IUYcy4Zu_1OnBl/edit?usp=drive_link&ouid=105479554793004732478&rtpof=true&sd=true) (template to be completed)

### Acceptance criteria

- [x] Initial Report has been reviewed by Product, Design and Dev, as necessary.
- [x] Remediation Plan completed
- [x] Identify any existing "accessibility" tickets in the backlog that would address these remediation items
- [x] Dev tickets created, as necessary, to execute on the remediation plan.

### Additional context

The 508 team have completed the 508 compliance review of the dotgov Registrar as requested via ticket  ICTR0044555.  

Attached is the 508 report outlining the 508 violations identified for dotgov Registrar. Based on the 508 assessment we have determined that remediation is required for this system/site.  

NEXT STEPS 

Please forward the defects to the development team to develop a plan for remediation. Afterward, please complete the attached CISA 508 Remediation Plan.doc and send it back to me. 

Once the issues are addressed by your team, please reply to this email with the updated Remediation Plan (there is no need to enter a new ACMS ticket). I will conduct a re-evaluation of the site to verify the previously identified issues are corrected and ensure no new defects are encountered.

RESOURCES 

ANDI Accessibility Testing Tool 
To expedite the remediation/retest process, your team’s developers should [add ANDI](https://www.ssa.gov/accessibility/andi/help/install.html) to their browser’s Favorite Bar (no admin rights needed).  This will allow them to perform quick 508 unit tests while making changes to fix the identified 508 defects. Unit testing will reduce the back and forth communication between our 508 Testing Team and the development team.  
Helpful Links  
•	[CISA Section 508 Program (sharepoint.com)](https://usdhs.sharepoint.com/sites/CISA_OCIO_Intranet/SitePages/CISA-Section-508-Program.aspx) 
•	[Technology needs of people who have disabilities (dhs.gov)](https://dhsconnect.dhs.gov/org/comp/mgmt/ocio/oast/playbook/technology-needs.html) 
•	[How to prioritize, monitor, and resolve Section 508 defects to minimize legal risk (dhs.gov)](https://dhsconnect.dhs.gov/org/comp/mgmt/ocio/oast/playbook/minimize-508-legal-risk.html)  
•	[How to obtain a Section 508 Compliance Determination (CDF) (dhs.gov)](https://dhsconnect.dhs.gov/org/comp/mgmt/ocio/oast/playbook/how-obtain-508-cdf.html) 
•	[How to address Section 508 in change control (dhs.gov)](https://dhsconnect.dhs.gov/org/comp/mgmt/ocio/oast/playbook/address-508-in-change-control.html) 
•	[How to provide an Alternative Means (dhs.gov)](https://dhsconnect.dhs.gov/org/comp/mgmt/ocio/oast/playbook/provide-alternative-means.html) 
•	[How to address Section 508 during operations & maintenance (dhs.gov)](https://dhsconnect.dhs.gov/org/comp/mgmt/ocio/oast/playbook/address-508-operations-and-maintenance.html)

### Links to other issues

Blocks:
- #2577

**Comment by PaulKuykendall** [comment]
Updated links in description to point back to Google Drive.

**Comment by AnnaGingle** [comment]
[Remediation plan](https://docs.google.com/document/d/1L7TW2RdxAFUrWWLSw3IUYcy4Zu_1OnBl/edit)
